packages/sw-routing/test/browser/end-to-end/sw/regexp-route.js

```diff
@@ -0,0 +1,25 @@
+/* global goog */
+
+importScripts('/packages/sw-routing/build/sw-routing.min.js');
+
+self.addEventListener('install', () => self.skipWaiting());
+self.addEventListener('activate', () => self.clients.claim());
+
+const routes = [];
+
+routes.push(new goog.routing.RegExpRoute({
+  regExp: new RegExp('static$'),
+  handler: {
+    handle: () => Promise.resolve(new Response('static response')),
+  },
+}));
+
+routes.push(new goog.routing.RegExpRoute({
+  regExp: new RegExp('echo/(\\w+)$'),
+  handler: {
+    handle: ({params}) => Promise.resolve(new Response(params[0])),
```

Comment:
Could you make this add two or three params and return it as JSON? Minor tweak but would be a slightly strong test.